Packages/com.unity.inputsystem/Tests/TestFixture/InputTestFixture.cs

```diff
@@ -140,6 +140,13 @@ public virtual void Setup()
                 InputSystem.settings.SetInternalFeatureFlag(InputFeatureNames.kUseOptimizedControls, true);
                 InputSystem.settings.SetInternalFeatureFlag(InputFeatureNames.kUseReadValueCaching, true);
                 InputSystem.settings.SetInternalFeatureFlag(InputFeatureNames.kParanoidReadValueCachingChecks, true);
+
+                // Default mock dialogs to avoid unexpected cancellation of standard flows
```

Comment:
Handy to keep testing flowing

Reply:
Or to set them and actually test the dialogs in the future which currently has zero test coverage from auto tests.